.github/workflows/build-release.yml

```diff
@@ -0,0 +1,92 @@
+# NOTE: multiple `make` commands could be replaced with
+# `make release` in future if all binaries are built on MacOS machine
+
+name: Build Release binaries
+
+on:
+  # Workflow executes when a new release is created
+  release:
+    types: [created]
+
+jobs:
+  # most binaries can be built on linux machine which is the most cost-efficient on GitHub actions
+  # for building darwin-arm64 binary we need Xcode, therefore, we need to build it on MacOS
+
+  linux-build:
+    name: Build binaries on Ubuntu
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Run Makefile
+        run: |
+          make package
+          make darwin
+          make linux
+          make linux-arm64
+          make windows
+          make windows32
+
+      # Artifacts docs: https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts
+      - name: Upload files as artifact
+        uses: actions/upload-artifact@v3
```

Comment:
is there a reason to not specify `retention-days: 1`?   Or do we need the default 90?